### PR TITLE
Margaux/local game screen

### PR DESCRIPTION
**Pull request by zwierski**
Pull request for the gamescreen that works with a popup. The popup cannot be tested yet as it requires robolectric which I'm unable to make work.

### CODE REVIEW

**Review by MaximeZmt — APPROVED** [review]
Some changes will be needed in the future, but not urgent. Good for now 👍

File: app/src/androidTest/java/ch/sdp/vibester/GamescreenActivityTest.kt [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        onView(withId(R.id.buzzersLayout)).check(matches(isDisplayed()))
+    }
+
+    @Test
+    fun scoresLayoutIsDisplayed() {
+        onView(withId(R.id.scoresTable)).check(matches(isDisplayed()))
+    }
+
+    // FIXME: Find a way to test the popup showing despite not being a component / make robolectric work
+/*
```
[the remark this answers is not in the document]
Comment (jiabaow) [reply app/src/androidTest/java/ch/sdp/vibester/GamescreenActivityTest.kt:49]: I think that's why she leaves the FIXME here

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -51,6 +53,18 @@
         app:layout_constraintTop_toTopOf="parent"
         app:layout_constraintVertical_bias="0.653" />
 
+    <Button
+        android:id="@+id/toLocalGameButton"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="L O C A L  G A M E"
```
Comment (jiabaow) [inline app/src/main/res/layout/activity_main.xml:60]: consider using @string instead of hardcode it

**Review by jiabaow — APPROVED** [review]
LGTM
The toast can be tested later in another PR as discussed

**Pull request merged**